docs/sources/alerting/set-up/configure-high-availability/_index.md

```diff
@@ -38,6 +38,31 @@ Since gossiping of notifications and silences uses both TCP and UDP port `9094`,
    By default, it is set to listen to all interfaces (`0.0.0.0`).
```

Comment:
what is set to listen to all interfaces? is there anything the user needs to do here before they get going?

Reply:
Not sure what you mean, but I don't think we need to extend on it.